### ISSUE TITLE
Typings are missing in package.json exports

### ISSUE THREAD

**Issue by PSanetra** [baumerdev/ibantools-germany#22]
On a new vite + typescript + react project I am encountering a build error:

```bash
> tsc -b && vite build

src/App.tsx:5:27 - error TS7016: Could not find a declaration file for module 'ibantools-germany'. '/home/projects/vitejs-vite-k3fube/node_modules/ibantools-germany/dist/esm/main.js' implicitly has an 'any' type.
  There are types at '/home/projects/vitejs-vite-k3fube/node_modules/ibantools-germany/dist/cjs/main.d.ts', but this result could not be resolved when respecting package.json "exports". The 'ibantools-germany' library may need to update its package.json or typings.

5 import {isValidIBAN} from "ibantools-germany";
                            ~~~~~~~~~~~~~~~~~~~


Found 1 error
```

You can reproduce the issue with this stackblitz: https://stackblitz.com/edit/vitejs-vite-k3fube?file=src%2FApp.tsx&terminal=build

I guess the fix needs to look similar to the solution in this comment: https://github.com/microsoft/TypeScript/issues/52363#issuecomment-1659179354

## Specifications

- Version: 1.2401.1

**Comment by baumerdev** [comment]
I just published version 1.2401.2 ; please check if it fixes your build.

**Comment by baumerdev** [comment]
I'm closing this. If this problem persists or reappears, please re-open this issue.
